subworkflows/nf-core/quantify_pseudo_alignment/tests/nextflow.config

```
@@ -0,0 +1,9 @@
+process {
+    withName: 'QUANTIFY_PSEUDO_ALIGNMENT:SE_GENE_UNIFIED' {
+        ext.args = '--assay_names counts,counts_length_scaled,counts_scaled,gene_lengths,tpm'
+    }
+
+    withName: 'QUANTIFY_PSEUDO_ALIGNMENT:SE_TRANSCRIPT_UNIFIED' {
+        ext.args = '--assay_names counts,lengths,tpm'
```

```suggestion
        ext.args = '--assay_names counts,transcript_lengths,tpm'
```

to be consistent with the gene object?

---

I'd actually go the other way- remove `gene` from `gene_lengths`. We know it's a gene object.